Romo/src/Drive/Telepresence/RMTelepresenceClientViewController.m

```diff
@@ -162,7 +162,7 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
                 break;
         }
         
-        [self.otManager.otSession signalWithType:@"tiltToAngle" data:@(angle) completionHandler:^(NSError *error) {}];
+//        [self.otManager.otSession signalWithType:@"tiltToAngle" data:@(angle) completionHandler:^(NSError *error) {}];
```

Comment:
Why were those commented out?

Reply:
These are part of OpenTok TelepresencePresence.